demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1453,6 +1439,46 @@ export class DemoMeetingApp
     this.initContentShareDropDownItems();
   }
 
+  async setAndGetEventReporter(configuration: MeetingSessionConfiguration): Promise<EventReporter> {
+    let eventReporter: EventReporter;
+    const ingestionURL = configuration.urls.eventIngestionURL;
+    if (ingestionURL) {
+      if (this.enableEventReporting) {
+        const eventReportingLogger = new ConsoleLogger('SDKEventIngestion', LogLevel.INFO);
+        const meetingEventClientConfig = new MeetingEventsClientConfiguration(
+          configuration.meetingId,
+          configuration.credentials.attendeeId,
+          configuration.credentials.joinToken
+        );
+        const eventIngestionConfiguration = new EventIngestionConfiguration(
+          meetingEventClientConfig,
+          ingestionURL
+        );
+        if (!['localhost', '127.0.0.1'].includes(location.hostname)) {
```

Comment:
We have multiple place where we do this check? Can you move it to its own routine so we could reuse it?

Reply:
Added an `isLocalHost` function and replaces two places where I got this while initializing the meeting session and setting up post logger for event reporter.

Reply:
Good callout!